web/js/pages/add_cron.js

```diff
@@ -1,23 +1,24 @@
-$(document).ready(function () {
-	$('#tabs').tabs();
+document.addEventListener('DOMContentLoaded', () => {
+	// TODO: Replace jQuery UI tabs with something else
+	$('.js-cron-tabs').tabs();
 
-	$('.cron-helper-container form').submit(function () {
-		$('#vstobjects input[name=v_min]')
-			.val($(this).find(':input[name=h_min]').val())
-			.effect('highlight');
-		$('#vstobjects input[name=v_hour]')
-			.val($(this).find(':input[name=h_hour]').val())
-			.effect('highlight');
-		$('#vstobjects input[name=v_day]')
-			.val($(this).find(':input[name=h_day]').val())
-			.effect('highlight');
-		$('#vstobjects input[name=v_month]')
-			.val($(this).find(':input[name=h_month]').val())
-			.effect('highlight');
-		$('#vstobjects input[name=v_wday]')
-			.val($(this).find(':input[name=h_wday]').val())
-			.effect('highlight');
+	const generateCronButtons = document.querySelectorAll('.js-generate-cron');
+	generateCronButtons.forEach((button) => {
+		button.addEventListener('click', () => {
+			const fieldset = button.closest('fieldset');
+			const inputNames = ['min', 'hour', 'day', 'month', 'wday'];
 
-		return false;
+			inputNames.forEach((inputName) => {
+				const value = fieldset.querySelector(`[name=h_${inputName}]`).value;
+				const formInput = document.querySelector(`#vstobjects input[name=v_${inputName}]`);
+
+				formInput.value = value;
+				formInput.classList.add('highlighted');
+
+				setTimeout(() => {
+					formInput.classList.remove('highlighted');
+				}, 500);
```

Comment:
I Don't mind removing or shorting this effect. 500 ms is personally a bit to long

Reply:
Sure, I've halved this timeout.